submission/launch.sh

```diff
@@ -1,19 +1,25 @@
-#!/bin/bash -ex
+#!/bin/bash
```

Comment:
The `-e` is good, you want the script to stop if anything fails. And `-x` is nice too.

Reply:
`-e` is somewhat good in this context. If the script stops if fails it's kind of bad because we get no data on `acs-interface`. The script failing is our problem, but if during `vagrant ssh` a command fails because the user sent a bad homework(e.g. bad `make run`), again no data sent to `acs-interface`. So I propose a compromise `set +e` before `vagrant ssh` and `set -e` after it. If something goes wrong let it be during `vagrant ssh` and we can give the poor soul the reason why.

Reply:
Yep, perfect. And you can write it like this so you don't have to `set -e` again:
```shell
(
  set +e
  vagrant ssh blah blah
)
```